### PR TITLE
[List] Removing MDCListThemer

### PR DESCRIPTION
**Pull request by galiak11**
# Description

Removing MDCListThemer from within our library.

# Issue

#9195

### CODE REVIEW

File: components/List/examples/MDCSelfSizingStereoCellExample.m [hunk 1]
```diff
@@ -30,7 +30,7 @@ @interface MDCSelfSizingStereoCellExample () <UICollectionViewDelegate, UICollec
 @property(nonatomic, strong) UICollectionView *collectionView;
 @property(nonatomic, strong) UICollectionViewFlowLayout *collectionViewLayout;
 @property(nonatomic, strong) NSArray *randomStrings;
-@property(nonatomic, strong) MDCListScheme *listScheme;
+@property(nonatomic, strong) MDCContainerScheme *containerScheme;
```
Comment (romoore) [inline components/List/examples/MDCSelfSizingStereoCellExample.m]: Nit: Please change to `id<MDCContainerScheming>` to match the catalog convention.

File: components/List/examples/MDCSelfSizingStereoCellExample.m [hunk 2]
```diff
@@ -40,7 +40,7 @@ - (void)viewDidLoad {
   [super viewDidLoad];
   self.parentViewController.automaticallyAdjustsScrollViewInsets = NO;
   self.automaticallyAdjustsScrollViewInsets = NO;
-  self.listScheme = [[MDCListScheme alloc] init];
+  self.containerScheme = [[MDCContainerScheme alloc] init];
```
Comment (romoore) [inline components/List/examples/MDCSelfSizingStereoCellExample.m]: Nit: Please check if it's not `nil` before assigning a value. The catalogs should be injecting a value to each example.

**Review by romoore — APPROVED** [review]

File: components/List/examples/MDCSelfSizingStereoCellExample.m (hunk 1 above)
Comment (galiak11) [reply components/List/examples/MDCSelfSizingStereoCellExample.m]: Done

File: components/List/examples/MDCSelfSizingStereoCellExample.m (hunk 2 above)
Comment (galiak11) [reply components/List/examples/MDCSelfSizingStereoCellExample.m]: done

**Comment by randallli** [comment]
We seemed to have missed the deprecation step and jumped right to delete. If this does not cause issues with the release we keep it in the release. 

Next time please follow https://github.com/material-components/material-components-ios/blob/develop/contributing/deprecation_policy.md

**Pull request merged**